Test/errortests/InvalidArgument1A.dfy

```diff
@@ -0,0 +1,2 @@
+// RUN: ! %baredafny verify %args --function-syntax:2 null.dfy > "%t"
```

Comment:
Can't you use %exit here as well?

Reply:
Yes - once this PR is approved, a sequel will roll it out to all the tests.